.semaphore/semaphore.yml

```diff
@@ -151,6 +155,81 @@ blocks:
           commands:
             - bats tests/sem_version_focal/${TEST}.bats
 
+  - name: Sem Version Tests Jammy ARM
+    dependencies:
+      - "Build local CLIs"
+    task:
+      agent:
+        machine:
+          type: r1-standard-4
+          os_image: ubuntu2204
+      prologue:
+        commands:
+          - wget https://github.com/semaphoreci/artifact/releases/download/v0.6.2/artifact_Linux_arm64.tar.gz -O /tmp/artifact_Linux_arm64.tar.gz
```

Comment:
Why do we need to download the artifact CLI here? Doesn't the `release/create.sh` script do it?

Reply:
There is no arm artifact and it is needed to pull artifacts for prologue.